src/App.jsx

```diff
@@ -72,127 +74,109 @@ function App() {
           task_version: version
         });
       }
-      setLogin(loggedIn);
+      setLogin(loggedIn)
     },
     [startDate]
-  );
+  )
 
   // Login logic
   useEffect(() => {
     // For testing and debugging purposes
-    console.log("Turk:", config.USE_MTURK);
-    console.log("Firebase:", config.USE_FIREBASE);
-    console.log("Prolific:", config.USE_PROLIFIC);
-    console.log("Electron:", config.USE_ELECTRON);
-    console.log("Video:", config.USE_CAMERA);
-    console.log("Volume:", config.USE_VOLUME);
-    console.log("Event Marker:", config.USE_EEG);
-    console.log("Photodiode:", config.USE_PHOTODIODE);
+    console.log('Turk:', config.USE_MTURK)
+    console.log('Firebase:', config.USE_FIREBASE)
+    console.log('Prolific:', config.USE_PROLIFIC)
+    console.log('Electron:', config.USE_ELECTRON)
+    console.log('Video:', config.USE_CAMERA)
+    console.log('Volume:', config.USE_VOLUME)
+    console.log('Event Marker:', config.USE_EEG)
+    console.log('Photodiode:', config.USE_PHOTODIODE)
     // If on desktop
     if (config.USE_ELECTRON) {
-      const { ipcRenderer } = window.require("electron");
-      setRenderer(ipcRenderer);
+      const { ipcRenderer } = window.require('electron')
+      setRenderer(ipcRenderer)
       ipcRenderer.send('updateEnvironmentVariables', config)
       // If at home, fill in fields based on environment variables
-      const credentials = ipcRenderer.sendSync("syncCredentials");
+      const credentials = ipcRenderer.sendSync('syncCredentials')
       if (credentials.envParticipantId) {
-        setEnvParticipantId(credentials.envParticipantId);
+        setEnvParticipantId(credentials.envParticipantId)
       }
       if (credentials.envStudyId) {
-        setEnvStudyId(credentials.envStudyId);
+        setEnvStudyId(credentials.envStudyId)
       }
-      setMethod("desktop");
-    }
-    // If online
-    else {
+      setMethod('desktop')
+    } else {
       // If MTURK
       if (config.USE_MTURK) {
         /* eslint-disable */
-        window.lodash = _.noConflict();
-        const turkId = getTurkUniqueId();
-        setPsiturk(new PsiTurk(turkId, "/complete"));
-        setMethod("mturk");
-        setLoggedIn(true, "mturk", turkId);
+        window.lodash = _.noConflict()
+        const turkId = getTurkUniqueId()
+        setPsiturk(new PsiTurk(turkId, '/complete'))
+        setMethod('mturk')
+        setLoggedIn(true, 'mturk', turkId)
         /* eslint-enable */
-      }
-
-      // If prolific
-      else if (config.USE_PROLIFIC) {
-        const pID = getProlificId();
+      } else if (config.USE_PROLIFIC) {
+        const pID = getProlificId()
         if (config.USE_FIREBASE && pID) {
-          setMethod("firebase");
-          setLoggedIn(true, "prolific", pID);
+          setMethod('firebase')
+          setLoggedIn(true, 'prolific', pID)
         } else {
-          setReject(true);
+          setMethod('online')
```

Comment:
I think this one should still be reject, local save + prolific doesn't make sense (no way to get the data from the participant)